cursive/vendor/index.py

```diff
@@ -1,15 +1,17 @@
-
-model_suffix_to_vendor_mapping = {
-    'openai': ['gpt-3.5', 'gpt-4'],
-    'anthropic': ['claude-instant', 'claude-2'],
-    'cohere': ['command'],
-    'replicate': ['replicate']
+model_prefix_to_vendor = {
+    "openai": ["gpt-3.5", "gpt-4"],
+    "anthropic": ["claude-instant", "claude-2"],
+    "cohere": ["command"],
+    "replicate": ["replicate"],
 }
 
+
 def resolve_vendor_from_model(model: str):
-    for vendor, suffixes in model_suffix_to_vendor_mapping.items():
-        if len([m for m in suffixes if model.startswith(m)]) > 0:
-            return vendor
+    if "/" in model:  # heuristic for OpenRouter.ai
```

Comment:
I feel like the best way to approach this is by setting OpenRouter as vendor, and resolving it when a user sets an API Key for it.

Will grab some time later today to implement this!